### PR TITLE
added config and test's for Viking2 as well as updated viking1 test's

### PR DESCRIPTION
**Pull request by gsn9**
This adds config file for viking2

solves: #23

### CODE REVIEW

File: SpiceQL/tests/QueryTests.cpp [hunk 1]
```diff
@@ -365,7 +366,29 @@ TEST_F(KernelDataDirectories, FunctionalTestSearchMissionKernelsViking1) {
   EXPECT_EQ(res["viking1"]["ck"]["reconstructed"]["kernels"].size(), 1);
   EXPECT_EQ(res["viking1"]["ck"]["smithed"]["kernels"].size(), 1);
   EXPECT_EQ(res["viking1"]["fk"]["kernels"].size(), 1);
-  EXPECT_EQ(res["viking1"]["iak"]["kernels"].size(), 1);
+  EXPECT_EQ(res["viking1"]["iak"]["kernels"].size(), 2);
   EXPECT_EQ(res["viking1"]["sclk"]["kernels"].size(), 2);
```
Comment (gsn9) [inline SpiceQL/tests/QueryTests.cpp:370]: updated viking 1 iak size after adding viking2 config

File: SpiceQL/tests/QueryTests.cpp [hunk 2]
```diff
@@ -365,7 +366,29 @@ TEST_F(KernelDataDirectories, FunctionalTestSearchMissionKernelsViking1) {
...
-  EXPECT_EQ(res["viking1"]["spk"]["reconstructed"]["kernels"].size(), 2);
+  EXPECT_EQ(res["viking1"]["spk"]["reconstructed"]["kernels"].size(), 3);
+}
```
Comment (gsn9) [inline SpiceQL/tests/QueryTests.cpp:372]: same for spk

File: SpiceQL/tests/QueryTests.cpp (hunk 1 above)
Comment (Kelvinrr) [reply SpiceQL/tests/QueryTests.cpp:370]: should viking1 conf be picking up a viking 2 kernel? Do they overlap?

Comment (gsn9) [reply SpiceQL/tests/QueryTests.cpp:370]: Yeah it seem's like there's overlapping between the two.

**Pull request merged**